src/pixie/images.nim

```diff
@@ -460,7 +460,7 @@ proc drawUber(
           if smooth:
             b.getRgbaSmooth(xFloat, yFloat)
           else:
-            b.getRgbaUnsafe(xFloat.round.int, yFloat.round.int)
+            b.getRgbaUnsafe(xFloat.int, yFloat.int)
```

Comment:
I think the rounding here is very important and it introduces tiny artifacts when not rounding. See the image_9_patch for rounding requirement.

Reply:
ah ok, I put it back